src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -383,7 +383,7 @@ public partial class Socket : System.IDisposable
         public int Send(System.ReadOnlySpan<byte> buffer, System.Net.Sockets.SocketFlags socketFlags) { throw null; }
         public int Send(System.ReadOnlySpan<byte> buffer, System.Net.Sockets.SocketFlags socketFlags, out System.Net.Sockets.SocketError errorCode) { throw null; }
         public bool SendAsync(System.Net.Sockets.SocketAsyncEventArgs e) { throw null; }
-        public void SendFile(string fileName) { }
+        public void SendFile(string? fileName) { }
```

Comment:
I thought we decided in https://github.com/dotnet/runtime/pull/32675#discussion_r382830596 that this should be non-null. Calling `SendFile(null)` won't do anything, so the assumption is you should pass in a fileName.

Reply:
If we are moving this to be nullable, to be consistent, we should annotate the `BeginSendFile` overload as nullable as well.

Reply:
We did, but issuing a warning doesn't actually solve any problems here, and it leads to an inconsistency with the other overload.  It's not harmful to pass null, it's always been allowed on .NET Framework, etc., so we may as well gain the consistency, avoid potential noisy warnings, and allow null.

Reply:
> If we are moving this to be nullable, to be consistent, we should annotate the BeginSendFile overload as nullable as well.

I thought I did.  I'll fix it.

Reply:
(The docs are also explicit about null being allowed.)